src/theory/arith/theory_arith.h

```diff
@@ -45,6 +45,10 @@ class TheoryArith : public Theory {
 
   TimerStat d_ppRewriteTimer;
 
+  // Used to prove pp-rewrites
+  EagerProofGenerator d_ppPfGen;
+  ProofNodeManager* d_pnm;
```

Comment:
The theory already has this pointer in the base class: https://github.com/CVC4/CVC4/blob/master/src/theory/theory.h#L237.

Reply:
Ah. Good point! Removed in b9332110c.